### DISCUSSION TITLE
How can I use this great thing now?

### DISCUSSION THREAD

**Discussion by GF-Huang** [dotnet/maui#310] [Q&A]
Where to download and install?

**Comment by LITTOMA** [comment]
No, you can't use it right now.

**Comment by Coding201212** [comment]
And when can we get the preview version？

**Comment by GeraudFabien** [comment]
Read the readme please : https://github.com/dotnet/maui#milestones

> Roadmap
> .NET MAUI and mobile SDK support will ship in concert with .NET 6. At present we do not have a shipping schedule for .NET 6


> Milestones
> - .NET MAUI previews Q4 2020 through Q3 2021
>    - Renderer architecture revisions
>    - Source solution and project simplification
>    - Complete approved proposals
>    - Implement MVU
> - .NET MAUI release candidate September 2021
> - .NET MAUI general availability November 2021

There should be preview soon but you may wait september. For a full feature preview.

**Comment by alfosua** [comment]
Is this current solution compilable, @GeraudFabien ?

**Comment by asheroto** [comment]
When NET 6 comes out with mobile support, will the UI be based on XAML?

**Comment by GeraudFabien** [comment]
@alfosuag  you may prefer compile Xamarin.form since it's not where improvement happen https://github.com/dotnet/maui/commits/main.
@asheroto  It's not related to this issue and please read the readme https://github.com/dotnet/maui

**Comment by mrwcjoughin** [comment]
Yes MAUI will have xaml support, just like Xamarin.Forms does.

MAUI is the successor to Xamarin.Forms

If you want to build mobile solutions with dot net already use
Xamarin.Forms and when MAUI comes out you can simply upgrade your projects
to MAUI

On Sat, 09 Jan 2021 at 13:09, Asher Oto <notifications@github.com> wrote:

> When NET 6 comes out with mobile support, will the UI be based on XAML?
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/dotnet/maui/issues/296#issuecomment-757133564>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ACOGERV7RXZU2DHP7U2AF4DSZA2OTANCNFSM4VWUZDRA>
> .
>
-- 
Thanks,

Matthew Joughin
Founder / CTO

+27 81 529 7169
http://www.adaptableapps.net

**Comment by asheroto** [comment]
Thank you!

**Comment by saint4eva** [comment]
> Yes MAUI will have xaml support, just like Xamarin.Forms does. MAUI is the successor to Xamarin.Forms If you want to build mobile solutions with dot net already use Xamarin.Forms and when MAUI comes out you can simply upgrade your projects to MAUI
> On Sat, 09 Jan 2021 at 13:09, Asher Oto ***@***.***> wrote: When NET 6 comes out with mobile support, will the UI be based on XAML? — You are receiving this because you are subscribed to this thread. Reply to this email directly, view it on GitHub <[#296 (comment)](https://github.com/dotnet/maui/issues/296#issuecomment-757133564)>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/ACOGERV7RXZU2DHP7U2AF4DSZA2OTANCNFSM4VWUZDRA> .
> -- Thanks, Matthew Joughin Founder / CTO +27 81 529 7169 http://www.adaptableapps.net

Would there be an excellent support for MVU C# UI?

**Comment by mrwcjoughin** [comment]
as far as i know yes MVU will be supported in MAUI

**Comment by saint4eva** [comment]
> as far as i know yes MVU will be supported in MAUI
> […](#)
> On Sat, Jan 16, 2021 at 8:54 AM saint4eva ***@***.***> wrote: Yes MAUI will have xaml support, just like Xamarin.Forms does. MAUI is the successor to Xamarin.Forms If you want to build mobile solutions with dot net already use Xamarin.Forms and when MAUI comes out you can simply upgrade your projects to MAUI On Sat, 09 Jan 2021 at 13:09, Asher Oto *@*.***> wrote: When NET 6 comes out with mobile support, will the UI be based on XAML? — You are receiving this because you are subscribed to this thread. Reply to this email directly, view it on GitHub <#296 (comment) <[#296 (comment)](https://github.com/dotnet/maui/issues/296#issuecomment-757133564)>>, or unsubscribe https://github.com/notifications/unsubscribe-auth/ACOGERV7RXZU2DHP7U2AF4DSZA2OTANCNFSM4VWUZDRA . -- Thanks, Matthew Joughin Founder / CTO +27 81 529 7169 http://www.adaptableapps.net Would there be an excellent support for MVU C# UI? — You are receiving this because you commented. Reply to this email directly, view it on GitHub <[#296 (comment)](https://github.com/dotnet/maui/issues/296#issuecomment-761518256)>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/ACOGERVXJWGLN5DYYNW2GY3S2EZZ7ANCNFSM4VWUZDRA> .

Okay.

**Comment by Cfun1** [comment]
Please close this as it is not an issue, it is more appropriate in [discussions](https://github.com/dotnet/maui/discussions) section.

**Comment by rodion-m** [comment]
So, is it still unknown when we can try a preview of MAUI?
